webapp/src/components/App/Singlestudy/explore/Modelization/Areas/Storages/StorageConfig/index.tsx

```diff
@@ -12,24 +12,26 @@
  * This file is part of the Antares project.
  */
 
+import { Box } from "@mui/material";
+import { useTranslation } from "react-i18next";
+import { useNavigate, useOutletContext, useParams } from "react-router-dom";
 import BackButton from "@/components/common/buttons/BackButton";
+import TabsView from "@/components/common/TabsView";
 import { getCurrentAreaId } from "@/redux/selectors";
 import { nameToId } from "@/services/utils";
 import type { StudyMetadata } from "@/types/types";
-import { Chip, Divider } from "@mui/material";
-import { useTranslation } from "react-i18next";
-import { useNavigate, useOutletContext, useParams } from "react-router-dom";
 import useAppSelector from "../../../../../../../../redux/hooks/useAppSelector";
+import AdditionalConstraints from "./AdditionalConstraints";
 import StorageForm from "./StorageForm";
-import StorageMatrices from "./StorageMatrices";
 
 function StorageConfig() {
   const { study } = useOutletContext<{ study: StudyMetadata }>();
   const navigate = useNavigate();
   const areaId = useAppSelector(getCurrentAreaId);
   const { storageId = "" } = useParams();
   const { t } = useTranslation();
-  const studyVersion = Number(study.version);
+  // const studyVersion = Number(study.version);
+  // TODO: conditionnal render of additional constraints only for studies > 9.2 ?
```

Comment:
dont know if it's in the specs, maybe we should render the additional constraints tab only for studies `>= 9.2` ?